app/features/conference/components/Conference.js

```diff
@@ -131,7 +147,104 @@ class Conference extends Component<Props, *> {
         setupWiFiStats(iframe);
 
         this._api.on('readyToClose', () => this._navigateToHome());
+
+        this._setName(this.props._name);
+        this._setEmail(this.props._email);
+        this._setAvatarURL(this.props._avatarURL);
```

Comment:
These may need to be set also in `componentWillReceiveProps`, in order to set the new avatar URL calculated after the email changed within the conference.

Reply:
Hmm. Avatar URL might need this.
Name and Email won't need it. I think we should hold this PR till we implement `emailChange` event so that we can debug code we will create with `componentWillReceiveProps`.
Otherwise, we might have to create another PR.

Reply:
Creating another PR is fine. Personally, I consider changing the email not to be a common use case. Moreover, even if we fix it today, it needs to be deployed, which won't happen immediately. So, let's write the code as correct was we can with these assumptions and once we fix the email thing we can check if this needs further fixing.